modules/@apostrophecms/doc-type/ui/apos/components/AposDocEditor.vue

```diff
@@ -67,11 +67,12 @@
               :conditional-fields="conditionalFields('other')"
               :doc-id="docId"
               :value="docFields"
-              @input="updateDocFields"
-              @validate="triggerValidate"
               :server-errors="serverErrors"
               :ref="tab.name"
               :generation="generation"
+              @input="updateDocFields"
+              @validate="triggerValidate"
+              @update-doc-data="onUpdateDocFields"
```

Comment:
This event is sent from `AposTemplateList` from Doc Template Library, and not anymore from the event bus.
Check this [comment](https://github.com/apostrophecms/doc-template-library/pull/22/files#r1268191545) in Doc Template Library